trulens_eval/examples/snowflake_demo/preload_apps.ipynb

```diff
@@ -25,13 +25,13 @@
     "load_dotenv()\n",
     "\n",
     "db_url = 'snowflake://{user}:{password}@{account}/{dbname}/{schema}?warehouse={warehouse}&role={role}'.format(\n",
-    "        user=os.environ['SF_USER'],\n",
-    "        account=os.environ['SF_ACCOUNT'],\n",
-    "        password=os.environ['SF_PASSWORD'],\n",
-    "        dbname=os.environ['SF_DB_NAME'],\n",
-    "        schema=os.environ['SF_SCHEMA'],\n",
-    "        warehouse=os.environ['SF_WAREHOUSE'],\n",
-    "        role=os.environ['SF_ROLE']\n",
+    "        user=os.environ['SNOWFLAKE_USER'],\n",
+    "        account=os.environ['SNOWFLAKE_ACCOUNT'],\n",
+    "        password=os.environ['SNOWFLAKE_PASSWORD'],\n",
```

Comment:
I think this is changed to `SNOWFLAKE_USER_PASSWORD`.

Reply:
updated